internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -442,20 +442,20 @@ func checkUintValueRange(valueType models.ValueType, val uint64) bool {
 	var isValid = false
 	switch valueType {
 	case models.Uint8:
-		if val >= 0 && val <= math.MaxUint8 {
```

Comment:
The code of checkUintValueRange function has been copied from edgex's [device-rest-go](https://github.com/edgexfoundry/device-rest-go/blob/cbc1fb126fca16d22f986b5cd5c305caa6710614/driver/resthandler.go#L324).
What about contributing this code to Edgex for easy maintenance?

Reply:
Thanks for the review. I do not mind to contribute this code. If you have permission, do it. I can't do it, because I need to start approval for this, and it will take some time.

Reply:
My recommendation is not urgent. I'm afraid that you code might be disappeared during next code refactoring with edgex.

Reply:
Well, I will try to make changes in the edgex

Reply:
@t25kim @MoonkiHong Done! Synchronization with the [edgexfoundry/device-rest-go](https://github.com/edgexfoundry/device-rest-go) project made, our changes were [applied](https://github.com/edgexfoundry/device-rest-go/pull/133)